Solutions/moritz_w/Lesson_4/eventlog.py

```diff
@@ -0,0 +1,88 @@
+"""
+Windows Event Log Parser
+Author: Moritz Wiesinger
+"""
+
+from datetime import datetime
+import unittest
+
+
+class LogEntry:
+    def __init__(self):
+        self.log_name = None
+        self.source = None
+        self.date = None
+        self.event_id = None
+        self.task = None
+        self.level = None
+        self.opcode = None
+        self.keyword = None
+        self.user = None
+        self.user_name = None
+        self.computer = None
+        self.description = []
+
+
+def parser(content):
+    """
+    Parses the content of a log.
+
+    :param content: List of event log lines
+    :type content: list[str]
+    :return: list[LogEntry]
+    """
+    ret = []
+    current_log_entry = None
+    for i in range(len(content)):
```

Comment:
You can use here function enumerate():
for i, line in enumerate(content):
   ...